src/server/services/country-codes/country-codes-iso-3166.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
As we need to use an approved list of country names, you'll need a map of country code to country name (from metadata.ts) and use that instead.  This one includes multiple names in an array for some countries which we can't support.